source/containers.tex

```diff
@@ -12202,6 +12192,10 @@
 
 \rSec3[flatset.cons.alloc]{Constructors with allocators}
 
+\pnum
+If \tcode{uses_allocator_v<container_type, Alloc>} is \tcode{false}
+the constructors in this subclause shall not participate in overload resolution.
+
```

Comment:
This removes the "if \tcode{Alloc} meets the allocator requirements as described in \iref{container.requirements.general}" part.  This was added after specific notes from LWG that it was needed.

Reply:
No other container or container-adaptor has such wording.
At the moment, I believe that if the user ever creates an `Alloc` such that `uses_allocator_v<container_type, Alloc>` is `true` but `Alloc` is NOT an allocator, then the user deserves what they get. Convince me otherwise? :)

Reply:
Look, I just work here, man.  I added that based on specific notes from Marshall and others at Batavia.  I think you need to convince *them*.

Reply:
To be explicit, Could you make a PR that has Alloc -> Allocator alone?  I'm fine doing this myself, but that would bork your other changes.

Reply:
Opened #15.